### ISSUE TITLE
Outdated version on crates.io

### ISSUE THREAD

**Issue by unixb0y** [ProbablyClem/cargo-todo#16]
When installing through the regular `cargo` command, it pulls the `v0.2.1` version which is severely outdated and might even have entirely wrong regexes, since they just don't match almost any 'spelling' of `//todo`.

**Comment by jayvdb** [comment]
Last published version is 3 years ago

Also recommend using https://github.com/taiki-e/upload-rust-binary-action

**Comment by sandtreader** [comment]
Indeed, the 'cargo install cargo-todo' version (which doesn't have a version number) installs this in ~/.cargo/todo_config:

```
^s*//s*todo\b
^s*//s*fix\b
^s*//s*fixme\b
```
         
Replace this with the regex from the README:

```
(?i)^\s*//\s*todo\b
(?i)^\s*//\s*fix\b
(?i)^\s*//\s*fixme\b
```

and it then approximately works, although it still doesn't recognise comments after code:

`    panic!("Lost my towel");        // TODO Don't panic`
